### DISCUSSION TITLE
Integration into the Controllable mod

### DISCUSSION THREAD

**Discussion by Chris6ix** [P3pp3rF1y/SophisticatedBackpacks#1064] [Ideas]
It would be amazing if SB had integration into the [Controllable mod](https://www.curseforge.com/minecraft/mc-mods/controllable), so that it's possible to add an open backpack shortcut into the radial menu. That would allow you to use SB's backpacks easily using a controller or when playing on the SteamDeck, like I am.

**Comment by Chris6ix** [comment]
Nevermind, I just found a way to do that.

**Comment by P3pp3rF1y** [comment]
yeah sounds to me like that mod should allow you to trigger any keybind in controls hence being able to open backpack as well
